utam-core/src/main/java/utam/core/selenium/appium/MobileDriverAdapter.java

```diff
@@ -75,7 +78,16 @@ final AppiumDriver switchToWebView(String title) {
     Set<String> contextHandles = appiumDriver.getContextHandles();
     for (String contextHandle : contextHandles) {
       if (!contextHandle.equals(NATIVE_CONTEXT_HANDLE)) {
-        AppiumDriver newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
+        AppiumDriver newDriver;
+        try {
+          newDriver = (AppiumDriver) appiumDriver.context(contextHandle);
+        } catch (WebDriverException e) {
+          warning("Context switch to webview '" + contextHandle + "' failed. Error : " + e.getMessage());
+          continue;
+        }
+        if (newDriver == null) {
```

Comment:
would suggest to use if (newDriver != null) {// do title check }

Reply:
updated